knpu/episode3/abstract-ship.md

```diff
@@ -4,15 +4,22 @@ There is one more thing that is special about the Rebel Ships. Since, they're
 the good guys we're going to give them some extra Jedi power. 
 
 Inside of `Ship` we have a `jediFactor` which is a value that is set from the
-database and a `getJediFactor` function. In the `BattleManager` this is used to 
-figure out if some super awesome Jedi powers are used during the battle. 
+database and a `getJediFactor` function:
```

Comment:
there were missing from the original transcription, but I prefer to always have functions with the `()`: `getJediFactor()`